extensions/documents/homie_last_updated_extension.md

```diff
@@ -0,0 +1,134 @@
+# Last-updated timestamp
+
+Version: **<!--VERSION-->1.0.0<!--VERSION-->**
+Date: **<!--DATE-->17. Feb 2020<!--DATE-->**
+Authors: **<!--AUTHORS-->Max Berger<!--AUTHORS-->**
+License: **<!--LICENSE-->[CCA 4.0](https://homieiot.github.io/license)<!--LICENSE-->**
+
+## Abstract
+This extension adds best-effort last-updated timestamps to properties.
+
+Homie devices will publish a last-updated timestamp along with any property value changes. The last-update timestamp can be sent by devices or gateways that do not constantly pull data from the real back-end. In this case, they can report when they last actually updated the data, so that controllers can act accordingly, and maybe ignore old data, or display this information in an UI.
```

Comment:
I would suggest to start with a rationale sentence or even section, something like

> In most Homie application scenarios it is implicitly assumed that a property update on MQTT directly follows the real world event, e.g., the press of a physical switch would trigger an MQTT message and a controller would react to said message and turn a light on.
>
> This extension addresses the special use case of a Homie device in which the timeliness of an event can't be guaranteed or in which the exact timestamp of a value is of importance.
> Examples for such use cases are
> * ...

Reply:
You sometimes write "last-update", sometimes "last-updated". I understand the incentive but this might be confusing.